model_security_adjust_oaw/views/sale_views.xml

```diff
@@ -13,6 +13,59 @@
             <field name="groups_id"
                    eval="[(6,0, [ref('base.group_sale_manager')])]"/>
         </record>
+        <!--Removing orignial "Sale" menu from External View-->
```

Comment:
Trying to remove the supplier group from the original "Sales" top menu item. Could not verify on dev-env due to prior security rules that might be still valid despite of that new rule.

Reply:
I suggest we should use 6 instead of 3 in this case.